Elmah.Io.Extensions.Logging/ElmahIoLoggerProvider.cs

```diff
@@ -7,16 +7,41 @@ public class ElmahIoLoggerProvider : ILoggerProvider
     {
         private readonly string _apiKey;
         private readonly Guid _logId;
+        private readonly FilterLoggerSettings _filter;
 
-        public ElmahIoLoggerProvider(string apiKey, Guid logId)
+        public ElmahIoLoggerProvider(string apiKey, Guid logId, FilterLoggerSettings filter = null)
         {
             _apiKey = apiKey;
             _logId = logId;
+
+            if (filter == null)
+            {
+                filter = new FilterLoggerSettings
+                {
+                    {"*", LogLevel.Trace}
```

Comment:
I think this is good for now. But in the future, I will probably change LogLevel.Trace to LogLevel.Warning. elmah.io is an error management system and in most scenarios it only makes sense to log warning, error and fatal messages. In some cases, having verbose logging in elmah.io as well, makes sense in order to see a trace of messages before an error happened etc.